src/dvclive/live.py

```diff
@@ -29,13 +32,6 @@
     open_file_in_browser,
 )
 
-try:
-    from dvc_studio_client.env import STUDIO_TOKEN
-    from dvc_studio_client.post_live_metrics import post_live_metrics
-except ImportError:
-    post_live_metrics = None
-    STUDIO_TOKEN = None
-
```

Comment:
Unrelated but `dvc_studio_client` has been in `install_requires` of dvclive for a while